src/dlgProfilePreferences.cpp

```diff
@@ -425,28 +425,23 @@ void dlgProfilePreferences::loadEditorTab()
     }
 
     // search engine load
-    // insert might be (/should?) moved elsewhere
-    mSearchEngineMap.insert("Bing", "https://www.bing.com/search?q=");
-    mSearchEngineMap.insert("DuckDuckGo", "https://duckduckgo.com/?q=");
-    mSearchEngineMap.insert("Google", "https://www.google.com/search?q=");
-
     // populate combobox
-    for(auto engineText : mSearchEngineMap.keys())
+    for(auto engineText : mpHost->mSearchEngineData.keys())
```

Comment:
No need to iterate here for ourselves,  grab all the keys (as a `QList<QString>`) and convert to a `QStringList` which can then populate the `QComboBox` in one _swell foop_:

    search_engine_combobox->addItems(QStringList(mpHost->mSearchEngineData.keys()));